### DISCUSSION TITLE
passwords visible in preview

### DISCUSSION THREAD

**Discussion by fdrab** [StackStorm/st2#5999] [UI]
Hello,

is it by design to have the fields marked "secret" visible in cleartext when pressing the "preview" button? I've found an old issue that stated this should already be resolved, but it's still present in 3.8.0.

Is there a setting I can mark to disable this?

BR,

Filip

**Comment by guzzijones** [comment]
please provide a short simplified example or describe your exact case in detail.

**Reply by fdrab** [reply]
created an issue in st2web: https://github.com/StackStorm/st2web/issues/1005

In the GUI, when about to execute an action or flow that has fields marked "secret", when clicking on the "Preview" button, any secrets are visible in cleartext.
